psi4/src/psi4/libmints/sieve.cc

```diff
@@ -56,10 +56,12 @@ void ERISieve::common_init() {
 
     Options &options = Process::environment.options;
     do_qqr_ = false;  // Code below for QQR was/is utterly broken.
+    do_csam_ = false;
```

Comment:
How do we switch this option on/off? I think we should write an accessor on the ERISieve objects and then have a setting so that this can be turned on/off at the JK (example) algorithm layer.

Reply:
I added a `do_csam` boolean as a constructor for the sieve object and an accessor method for the variable